### ISSUE TITLE
Include missing names, roles, values

### ISSUE THREAD

**Issue by marcossanhudo** [marcossanhudo/business-appointment-app#22]
## The problem

The W3C recommends, in [WCAG success criterion 4.1.2](https://www.w3.org/TR/WCAG22/#name-role-value), that all elements have a name, role, and value, which is currently not done on the front-end.

## The solution

Include all missing information for names and roles, and expose all values to assistive technology. Refer to component prototypes.

## Related documentation

[WAI-ARIA Understanding Docs](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).
